taxonomies/ingredients_processing.txt

```diff
@@ -908,6 +911,7 @@ en:semi candied, semi-candied
 # <en:unsweetened
 de:ungezuckert
 
+en:oiled
```

Comment:
The test probably failed because I added "en:oiled" but I don't know why that is a problem.

Reply:
The test expects German entry to be the main entry: https://github.com/openfoodfacts/openfoodfacts-server/blob/dcb9640ea1490429a38281a57c5d14dd202111fe/t/ingredients_processing.t#L425-L439

I'd ping @aleene in case there's a reason why it doesn't have English entry.

Reply:
In English, I don't see any ingredients that are "oiled": https://us.openfoodfacts.org/ingredients?filter=oiled
So I guess it's fine to add it in English, there wouldn't be false positives.

What about the German ones with geölt, geölte, geölter, geöltes, what kind of ingredients are those?

Reply:
in another PR, we should add "boiled", "parboiled", "charbroiled", "flamebroiled" though :)

Reply:
I know that raisins are sometimes oiled but haven't found any ingredients containing "geölt".